common/utils/format-date.ts

```diff
@@ -1,15 +1,5 @@
-import moment, { Moment } from 'moment';
-import 'moment-timezone';
 import { isFuture, isPast, isSameDay } from './dates';
 
-type DateObj = { M?: number; Y?: number };
-
-export type DateTypes = Date | string | Moment | DateObj;
-
-export function london(d?: DateTypes): Moment {
-  return moment.tz(d, 'Europe/London');
-}
```

Comment:
As discussed elsewhere, eventually I'd like to ditch this function entirely, because I think it covers up a lot of holes in our type system (e.g. when a `string` gets passed as a `Date`). I haven't quite gotten it out yet, but it's only used in `@weco/catalogue` now so I've moved it there.